### PR TITLE
Add specific questions

### PR DESCRIPTION
**Pull request by Juhantam**
I will also add the Anwser class later, but this should be pretty decent for testing. Let me know if I should change anything.

### CODE REVIEW

File: src/main/java/general/questions/ImageQuestion.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package general.questions;
+
+import java.nio.file.Path;
+import java.util.List;
+
+public class ImageQuestion extends BaseQuestion implements Question {
+
+
```
Comment (Juhantam) [inline src/main/java/general/questions/ImageQuestion.java]: Oh whoops, I must've accidentally removed the Path object, I'll fix it once I start working on the anwser stuff.

File: src/main/java/general/questions/AudioQuestion.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    public String getAddress() {
+
+        return address;
+
+    }
+
+    public AudioQuestion(QuestionType questionType) {
+
+        super(questionType);
```
Comment (TanelOrumaa) [inline src/main/java/general/questions/AudioQuestion.java]: Shouldn't this have questiontype from Enum instead from input? You already know you are creating audio question.

File: src/main/java/general/questions/AudioQuestion.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+    }
+
+    public AudioQuestion(QuestionType questionType, boolean scoreDegradation, String question, List<String> correctAnswers, int potentialPoints, int time) {
```
Comment (TanelOrumaa) [inline src/main/java/general/questions/AudioQuestion.java]: Same with this constructor.

File: src/main/java/general/questions/ImageQuestion.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    public ImageQuestion(QuestionType questionType) {
```
Comment (TanelOrumaa) [inline src/main/java/general/questions/ImageQuestion.java]: Same comment here as in the audio question.

**Pull request merged**